src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
```

Comment:
move it to the `*` selector
```suggestion
```

Reply:
but according to BEM rules the '*' selector cannot be used...